stable/aws-s3-proxy/README.md

```diff
@@ -8,37 +8,11 @@ See here for configuration via environment variables: https://github.com/pottava
 
 **Homepage:** <https://github.com/pottava/aws-s3-proxy>
 
-## How to install this chart
-
-Add Delivery Hero public chart repo:
```

Comment:
Why is all this being removed?

Reply:
that's what I'm trying to figure out, the helm-docs command removed it just because.